packages/bff/src/run-migration.ts

```diff
@@ -1,20 +1,19 @@
-import util from 'node:util';
 import 'reflect-metadata';
-import { initAppInsightWithRetry } from './azure/ApplicationInsightsInit.ts';
+import { intitialize } from './azure/ApplicationInsights.ts';
 import config from './config.ts';
 import { connectToDB } from './db.ts';
 
 export const runMigrationApp = async () => {
   // App Insight setup
-  if (config.applicationInsights.connectionString) {
+  if (config.applicationInsights.enabled) {
     try {
-      await initAppInsightWithRetry(config.applicationInsights.connectionString, 10);
+      intitialize();
     } catch (e) {
```

Comment:
trenger vi å catche, logge feil og så force-exit med (1) .... vs å la catchen gå ubehandlet hen?

Reply:
gjortde om litt, gjerne sjekk igjen!

Reply:
helt enig forøvrig